inst/extdata/aglu/A_agSupplySector.csv

```diff
@@ -0,0 +1,22 @@
+# Title: A_agSupplySector
+# File: A_agSupplySector.csv
+# Description: agSupplySectors - name / market / unit
+# Original source: NA
+# Units: NA
+# ----------
+AgSupplySector,output.unit,input.unit,price.unit,calPrice,market,logit.exponent,logit.type
+Corn,Mt,thous km2,1975$/kg,,global,-3,
+FiberCrop,Mt,thous km2,1975$/kg,,global,-3,
+FodderGrass,Mt,thous km2,1975$/kg,,regional,-3,
+FodderHerb,Mt,thous km2,1975$/kg,,global,-3,
+MiscCrop,Mt,thous km2,1975$/kg,,global,-3,
+OilCrop,Mt,thous km2,1975$/kg,,global,-3,
+OtherGrain,Mt,thous km2,1975$/kg,,global,-3,
+PalmFruit,Mt,thous km2,1975$/kg,,global,-3,
+Rice,Mt,thous km2,1975$/kg,,global,-3,
+Root_Tuber,Mt,thous km2,1975$/kg,,global,-3,
+SugarCrop,Mt,thous km2,1975$/kg,,global,-3,
+Wheat,Mt,thous km2,1975$/kg,,global,-3,
+Forest,billion m3,thous km2,1975$/m3,,global,-3,
+biomass,EJ,thous km2,1975$/GJ,,regional,-3,
+Pasture,Mt,thous km2,1975$/kg,,regional,-3,
```

Comment:
Actually I added this file because it was listed as input, but it never actually gets used anywhere. I took references to it out of the chunk, but I should probably just drop this file entirely.  (I double checked the old DS chunk; it's read in as an input and then never used).